atomic_reactor/pulp_util.py

```diff
@@ -103,7 +103,7 @@ def _set_auth(self):
 
     def _ensure_repos(self, pulp_repos, repo_prefix):
         repos = pulp_repos.keys()
-        found_repos = self.p.getRepos(repos, fields=["id"])
+        found_repos = self.p.getRepos(repos, fields=["id"], distributors=True)
```

Comment:
Well, hold on. This now won't work with versions of dockpulp that don't allow that parameter in getRepos(). :(

I think we need to catch `TypeError` and try again in that case, e.g.
```
kwargs = {'fields': ['id'], 'distributors': True}
try:
    found_repos = ...(..., **kwargs)
except TypeError:
    # dockpulp < x.y doesn't support getRepos(..., distributors=...)
    del kwargs['distributors']
    found_repos = ...(..., **kwargs)
```